app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Api::V1::CommentsController < ApplicationController
+  before_action :find_user_and_post
+  skip_before_action :verify_authenticity_token, only: [:create]
+  skip_before_action :authenticate_user!, except: []
+
+  def index
+    @comments = @post.comments
+    render json: @comments
+  end
+
+  def create
+    @comment = @post.comments.new(comment_params)
+    @comment.author = @user
+
+    if @comment.save
+      render json: @comment, status: :created
+    else
+      render json: { error: @comment.errors.full_messages }, status: :unprocessable_entity
+    end
+  end
```

Comment:
- [ ] Great job creating the controllers for the API endpoints defining the various actions in the Comments and Posts controllers and implementing the logic. :clap:  However, I noticed that you didn't define the corresponding routes for the API endpoints in your `routes.rb file`. As a result, I was unable to test your API endpoints to check if they behave as expected :+1: 
- [ ] I got this error when I tried to access each of the required endpoints :+1: 
- Error Message
![image](https://github.com/kayjeee/Hi-Blog_app_kayjeee/assets/102699958/bff62920-9149-4a34-bf70-64bcc14e2304)

- [ ] So kindly create API endpoints for your routes, and ensure that you specify which controller actions will handle the HTTP requests for those routes in your `routes.rb` file. :100: 
- Example Implementation
```ruby
    # Define API routes under a separate namespace
    namespace :api do
      namespace :v1 do
        resources :users, only: [:show, :index] do
          resources :posts, only: [:index] do
            resources :comments, only: [:index, :create] do
            member do
              post 'like'
              delete 'unlike'
            end
            end
          end
        end
      end
    end
```